packages/sources/layer2-sequencer-health/src/endpoint/health.ts

```diff
@@ -7,19 +7,20 @@ import {
   util,
 } from '@chainlink/ea-bootstrap'
 import { ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
-import { ExtendedConfig, Networks } from '../config'
+import { EVMNetworks, ExtendedConfig, Networks } from '../config'
 import {
   requestBlockHeight,
   getSequencerHealth,
   NetworkHealthCheck,
   getStatusByTransaction,
 } from '../network'
+import { checkStarkwareSequencerPendingTransactions } from '../starkware'
 
-export const supportedEndpoints = ['health']
+export const supportedEndpoints = ['health', 'health-evm']
```

Comment:
Why did we add this in? It doesn't seem used.

Reply:
ah this was leftover.  Just removed this.